docs/manual/getting_started.rst

```diff
@@ -1987,29 +2172,29 @@ next subsection).
 To exchange data, applications' business logic with Cyclone DDS must:
 
 1. Declare its subscription and involvement into a **DDS domain**. A
-   DDS domain is an administrative boundary that defines, scopes and
+   DDS domain is an administrative boundary that defines, scopes, and
    gathers all the DDS applications, data, and infrastructure that needs
    to interconnect and share the same data space. Each DDS domain has a
    unique identifier. Applications declare their participation within a
    DDS domain by creating a **Domain Participant entity**.
-2. Create a **Data topic** that has the data type described in the data
-   model. The data types define the structure of the Topic. The Topic is
-   therefore an association between the topic name and datatype. QoSs
-   can be optionally added to this association. A Topic therefore
-   categories the data in logical classes and streams.
-3. Create at least a **Publisher** , a **Subscriber**, and **Data
-   Readers** and **Writers** objects that are specific to the topic
+2. Create a **Data topic** with the data type described in the data
+   model. The data types define the structure of the Topic. The Topic is,
+   therefore, an association between the topic name and datatype. QoSs
+   can be optionally added to this association. A Topic, therefore
+   categorizes the data into logical classes and streams.
+3. Create at least a **Publisher**, a **Subscriber**, and **Data
+   Readers** and **Writers** object specific to the topic
    created earlier. Applications may want to change the default QoSs at
    this stage. In the Hello world! example, the ``ReliabilityQoS`` is
    changed from its default value (Best-effort) to Reliable.
-4. Once the previous DDS computational object s are in place, the
+4. Once the previous DDS computational objects are in place, the
    application logic can start writing or reading the data.
 
-At the application level, readers and writers do not need to be aware of
-each other. The reading application, hereby designated as application
-Subscriber polls the data reader periodically, until a writing
-application, hereby called application Publisher, provides the required
-data into the shared topic, namely ``HelloWorldData_Msg``.
+At the application level, readers and writers need not be aware of
+each other. The reading application, now designated as application
+Subscriber polls the data reader periodically until a writing
+application, as a result of this called application Publisher, 
```

Comment:
I have provided some fixes for this. However, I think this whole paragraph needs some tweaking because of how run-on it is.
```suggestion
each other. The reading application, now designated as application
Subscriber, polls the data reader periodically until a writing
application, designated as application Publisher, 
```